docs/pages/experiments/slider-range-aria-label.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import * as Slider from '@base_ui/react/Slider';
+import { useSliderContext } from '@base_ui/react/Slider';
+import { Styles, TrackFill } from './slider';
+
+function Label(props: any) {
+  const { id: idProp, ...otherProps } = props;
+  const defaultId = React.useId();
+  const labelId = idProp ?? defaultId;
+
+  const { subitems } = useSliderContext();
+
+  const htmlFor = Array.from(subitems.values())
+    .reduce((acc, item) => {
+      return `${acc} ${item.inputId}`;
+    }, '')
+    .trim();
+
+  return <label id={labelId} htmlFor={htmlFor} {...otherProps} />;
+}
+
+function LabelRange(props: any) {
+  const { id: idProp, ...otherProps } = props;
+
+  const defaultId = React.useId();
+  const labelId = idProp ?? defaultId;
+
+  return <span id={labelId} {...otherProps} />;
+}
+
+export default function App() {
```

Comment:
> Adding aria-label=“start” and aria-label=“end” to the range slider inputs might help with SR announcements

We could set this as the default `aria-label` if none is explicitly provided as a prop, but only for ranges with 2 values, is that too fancy? 😓 @colmtuite 

Demo: https://deploy-preview-373--base-ui.netlify.app/experiments/slider-range-aria-label